### ISSUE TITLE
ACC-260.2 (Level AA) _ Modify Modules _ Navigation menu _ Truncation of header menu 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3136]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Navigation menu
Affected Check Point:	ACC-260.2
Short text:	ACC-260.2 (Level AA) _ Modify Modules _ Navigation menu _ Truncation of header menu 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Now add the Device with Text Resize settings of width 1280 CSS pixels and height 1024 CSS pixels from the ‘Developers tool’ from the ‘Chrome’ Browser and apply the settings . Zoom the page to 200% in the browser.
•	Now refresh the page after the settings are applied.

3. Issue Type: Manual
Observed Behavior:
On resizing the text to 200% zoom, it is observed that there is truncation of the navigation menu i.e. ‘garden-kyma-stage…’ within the main header shell bar region.
Screenshot:
Default:
 
After text resize to 200% zoom:
 

Expected Behavior:
Truncations should be avoided and all the UI elements should be clearly visible when text is resize up to 200% zoom. The screen layout must be scaled so that there is no loss either content or function when text is resized to 200% zoom.

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-260.2 (Level AA) _ Modify Modules _ Navigation menu _ Truncation of header menu.docx](https://github.com/user-attachments/files/16659549/ACC-260.2.Level.AA._.Modify.Modules._.Navigation.menu._.Truncation.of.header.menu.docx)
[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16659550/Issue.processing-09.2021.002.docx)

**Comment by mrCherry97** [comment]
Work as expected. UI5 implementation is covering this functionality.
